### PR TITLE
Integrate Sonarqube

### PR DESCRIPTION
**Pull request by SathyaRupa**
- Setup sonarqube in the local environment

### CODE REVIEW

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -36,3 +36,29 @@ Docker-compose
...
+
+## Steps to setup SonarQube locally
+
+- run `make docker`
+- Hit this link: http://localhost:9000/
+- Create a local project. name: Airport, project key : Airport
+- Change `main` to `master`
+- Select an option of your choice for the second step
+- Click create project.
+- Select locally
```
Comment (vraravam) [inline CONTRIBUTING.md]: formatting.

**Review by vraravam — CHANGES_REQUESTED** [review]
please include images for each step

File: CONTRIBUTING.md (hunk 1 above)
Comment (SathyaRupa) [reply CONTRIBUTING.md]: Made the changes

**Comment by SathyaRupa** [comment]
> please include images for each step

Added images for most steps

File: CONTRIBUTING.md [hunk 2]
```diff
@@ -36,3 +36,30 @@ Docker-compose
 - Access the running app on [local](https://0.0.0.0:8080/)
 
 -Access the Jaeger UI in [local](http://localhost:16686)
+
+## Steps to setup SonarQube locally
+
+- run `make docker`
+- Hit this [link](http://localhost:9000/) [Credentials --> Username : Admin, Password : Admin]
+- Create a local project with name as `Airport` and project key as `Airport` [Image](https://drive.google.com/file/d/1MTOFer7h9epf3UPkkE3qdIkVKaCG6bhb/view?usp=drive_link)
```
Comment (vraravam) [inline CONTRIBUTING.md]: Use relative path for the image. And store the image in this repo itself. If you store in a google drive, a new joinee will then have to request access to that file/folder, correct?

**Review by vraravam — CHANGES_REQUESTED** [review]

File: CONTRIBUTING.md (hunk 2 above)
Comment (SathyaRupa) [reply CONTRIBUTING.md]: Made changes. Moved the images to repo, and used them instead of drive.

**Review by vraravam — APPROVED** [review]

**Review by vraravam — APPROVED** [review]

**Pull request merged**